src/API/Reports/Export/Controller.php

```diff
@@ -159,8 +159,11 @@ public function export_items( $request ) {
 		$report_type = $request['type'];
 		$report_args = empty( $request['report_args'] ) ? array() : $request['report_args'];
 		$send_email  = isset( $request['email'] ) ? $request['email'] : false;
-		$export_id   = str_replace( '.', '', microtime( true ) );
-		$total_rows  = ReportExporter::queue_report_export( $export_id, $report_type, $report_args, $send_email );
+
+		$default_export_id = str_replace( '.', '', microtime( true ) );
+		$export_id         = apply_filters( 'woocommerce_admin_export_id', $default_export_id );
```

Comment:
I think it would be good to validate `$export_id` is a `string` in case something else is returned by the filter. It might also need to be sanitized, e.g. to prevent a [path traversal attack.](https://owasp.org/www-community/attacks/Path_Traversal)